sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListenerSuite.scala

```diff
@@ -754,8 +754,6 @@ class SQLAppStatusListenerSuite extends SharedSparkSession with JsonTestUtils
       SparkPlanGraph(newPlan)
         .allNodes.flatMap(_.metrics.map(_.accumulatorId))
 
-    // Assume that AQE update sparkPlanInfo with newPlan
-    // ExecutionMetrics will be replaced using newPlan's SQLMetrics
```

Comment:
can we keep the comment and use `appended` instead of `replaced`?

Reply:
done